addons/web/static/tests/views/fields/one2many_field_tests.js

```diff
@@ -12047,7 +12047,7 @@ QUnit.module("Fields", (hooks) => {
         }
     );
 
-    QUnit.test(
+    QUnit.skip(
```

Comment:
I see a single occurrence of this test failing in the runbot build errors, is it really necessary to skip it?